virttest/test_setup.py

```diff
@@ -2423,38 +2424,71 @@ def __init__(self, test, log_level="1", log_file=""):
         self.log_level = log_level
         self.log_file = log_file
         self.test = test
-        self.libvirtd = utils_libvirtd.Libvirtd()
-        self.libvirtd_conf = utils_config.LibvirtdConfig()
-        self.backupfile = "%s.backup" % self.libvirtd_conf.conf_path
+        self.daemons_dict = {}
+        self.daemons_dict["libvirtd"] = {
+            "daemon": utils_libvirtd.Libvirtd("virtqemud"),
+            "conf": utils_config.LibvirtdConfig(),
+            "backupfile": "%s.backup" % utils_config.LibvirtdConfig().conf_path}
+        if (libvirt_version.version_compare(5, 6, 0) and
+           libvirtd_decorator.get_libvirtd_split_enable_bit()):
```

Comment:
To detect modular daemon environment, I suggest we use a single function instead of checking version and enable bit everywhere. But anyway we can leave it as is for now.

Reply:
Yes, I agree. We still need to update get_libvirtd_split_enable_bit() later, may be at that time, we can do it.